build.sbt

```diff
@@ -52,8 +52,15 @@ lazy val remoteServer = Project("remote-server", file("remote-server"))
   .settings(Testing.settings: _*)
   .settings(Assembly.remoteServerAssemblySettings: _*)
 
+lazy val piClient = Project("pi-client", file("pi-client"))
```

Comment:
is this really needed? just lets the request flow through. I suppose it is not needed at all.
he?